assets/src/components/map.tsx

```diff
@@ -371,6 +378,56 @@ const Autocenterer = ({
   return <></>
 }
 
+const garageLeafletIcon = Leaflet.divIcon({
+  html: garageIcon,
+  className: "m-garage-icon",
+})
+
+const Garage = ({
+  garage,
+  zoomLevel,
+}: {
+  garage: Garage
+  zoomLevel: number
+}) => (
+  <>
+    <Marker
+      interactive={false}
+      key={garage.name}
+      position={[garage.lat, garage.lon]}
+      icon={garageLeafletIcon}
+    />
+    {zoomLevel >= 16 && (
+      <Marker
+        interactive={false}
+        position={[garage.lat, garage.lon]}
+        icon={Leaflet.divIcon({
+          className: "m-garage-icon__label",
+          html: `<svg height="30" width="200">
+                    <text x="22" y="15">${garage.name}</text>
+                  </svg>`,
```

Comment:
question: why use inline svg here?

Reply:
I was following the pattern of the existing [vehicle icon label](https://github.com/mbta/skate/blob/9addac1563f9fa0ea646b1efcd60d2ecbcaa0df1/assets/src/components/map.tsx#L94). Though I haven't done. much with SVGs before so if there is a alternative preferred way I handling I'm interested in learning!

Reply:
@firestack thanks for digging that old PR up!

Reply:
https://leafletjs.com/reference.html#divicon-html says it's `String|HTMLElement` so I'd assume that we could return an element fragment from JSX to this?

Reply:
Oh, interesting, I saw an older page that just said that it had to be a string. Yeah if passing an `HTMLElement` works that would be great.

Reply:
This may be my own inexperience around `HTMLElement` - my first thought was that I would only need to remove the quotations around the SVG and it would just work:tm:, though it is in fact a ReactElement & doesn't overlap with `HTMLElement`. From how I've seen `HTMLElement`s created before, it involves calling `document.createElement` and then eventually `.innerHtml`, which takes a string and therefore doesn't seem that different from the current approach. 

Is there a terser way to create this svg element as an `HTMLElement` that would be preferable to the current approach?

Reply:
Wow I didn't expect JSX to make it so difficult to go from `FunctionComponentElement` to `HTMLElement` lol!

Reply:
Same here 😄

Reply:
I just want to say that I think this is fine as is, especially given this is something we do elsewhere, I'm a bit sad that JSX can't handle this case for us, but maybe we can address that somehow in some future work.